### DISCUSSION TITLE
v1.12.2

### DISCUSSION THREAD

**Discussion by FlechazoW** [DTStack/chunjun#659] [General]
修复已知bug；
增加inceptor插件和influxdb；

<hr /><em>This discussion was created from the release <a href='https://github.com/DTStack/chunjun/releases/tag/'>v1.12.2</a>.</em>

**Comment by wujian9407** [comment]
![image](https://user-images.githubusercontent.com/108928562/177962018-0021dd10-e928-4af8-8705-8d9f1ca8fcaf.png)
老师，您好这个报错是什么原因

**Reply by FlechazoW** [reply]
在项目的bin目录下有个install_jars的脚本，本地先执行下即可。

**Reply by FlechazoW** [reply]
另外，你这应该重新开一个discussion 或者 issue来提问。
